components/brave_shields/browser/brave_shields_util_unittest.cc

```diff
@@ -42,64 +42,44 @@ TEST_F(BraveShieldsUtilTest, GetPatternFromURL) {
   auto pattern = GetPatternFromURL(GURL());
   EXPECT_EQ(ContentSettingsPattern::Wildcard(), pattern);
 
-  // no scheme wildcard
+  // scheme is a wildcard, sould match any scheme
```

Comment:
typo

Reply:
Fixed in bdc2851b5386b1ae42a134550a217edb6267ef57